test/unit/option_spec.lua

```diff
@@ -0,0 +1,61 @@
+local helpers = require("test.unit.helpers")
+
+local to_cstr = helpers.to_cstr
+local eq      = helpers.eq
+
+local option = helpers.cimport("./src/nvim/option.h")
+local globals = helpers.cimport("./src/nvim/globals.h")
+
+local check_ff_value = function(ff)
+  return option.check_ff_value(to_cstr(ff))
+end
+
+local get_sts_value = function()
+  return option.get_sts_value()
+end
+
+describe('check_ff_value', function()
+
+  it('views empty string as valid', function()
+    eq(1, check_ff_value(""))
+  end)
+
+  it('views "unix" as valid', function()
+    eq(1, check_ff_value("unix"))
+  end)
+
+  it('views "dos" as valid', function()
+    eq(1, check_ff_value("dos"))
+  end)
+
+  it('views "mac" as valid', function()
+    eq(1, check_ff_value("mac"))
+  end)
```

Comment:
The various "valid" values are probably too granular as individual tests. Consider combining them into one test.